modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/SidePanelContent.tsx

```diff
@@ -27,6 +27,7 @@ export function saveAndReload() {
 }
 
 export function openToast(options: {
+	autoClose?: number;
```

Comment:
No need to upgrade this function, as we should directly use the one from `'frontend-js-web'`.

Reply:
I agree, done!